civis/io/_tables.py

```diff
@@ -115,6 +156,20 @@ def read_civis(table, database, columns=None, use_pandas=False,
     if client is None:
         # Instantiate client here in case users provide a (deprecated) api_key
         client = APIClient(api_key=api_key, resources='all')
+
+    if use_pandas == True:
+        if dtype == 'from_table':
+            table_id = client.get_table_id(table=table, database=database)
+            py_types, date_cols = _redshift_to_py(client, table_id)
+            pd_kwargs = {'parse_dates': date_cols,
+                         'dtype': py_types,
+                         'engine': 'python',
+                         'infer_datetime_format': True,
+                         'encoding': 'utf8'}
+            kwargs.update(pd_kwargs)
+        else:
+            kwargs['dtype'] = dtype
```

Comment:
Will this work if the user passes `columns`?  I think it will at least break if the users passes sql into the columns.

Reply:
It would not work with columns in general no.